qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -167,6 +167,24 @@ def convert_frame_change(self, shift, instruction):
         }
         return self._qobj_model(**command_dict)
 
+    @bind_instruction(commands.SetChannelFrequencyInstruction)
+    def convert_set_channel_frequency(self, shift, instruction):
+        """ Return converted `SetChannelFrequencyInstruction`.
+
+        Args:
+            shift (int): Offset time.
+            instruction (SetChannelFrequencyInstruction): set channel frequency instruction.
+        Returns
+            dict: Dictionary of required parameters.
+        """
+        command_dict = {
+            'name': 'scf',
```

Comment:
here its worth being verbose, and making it `'set_frequency'`, since this is the only context the backend will receive about the command

Reply:
Fixed in 9b85ff8

Reply:
> here its worth being verbose, and making it 'set_frequency', since this is the only context the backend will receive about the command

I am not so certain I agree, this increases the qobj size and is not akin to an assembly level calling convention. The current convention for the qobj, `pv`, `fc` are short, whereas `acquire` violates this. I feel the same way about `parameter_pulse` being too verbose.

Reply:
So stick with `scf` both as prefix in the command and in the converter?

Reply:
I think that makes the most sense

Reply:
I ended up going with `sf` to properly reflect the name of the command.